scripts/spack/packages/serac/package.py

```diff
@@ -141,7 +141,7 @@ class Serac(CachedCMakePackage, CudaPackage):
     # Libraries that do not have a debug variant
     depends_on("conduit@0.7.2serac~shared~python~test")
     depends_on("adiak@0.2.1~shared+mpi", when="+profiling")
-    depends_on("caliper@master~shared+mpi+adiak~papi", when="+profiling")
+    depends_on("caliper@2.7.0~shared+mpi+adiak~papi", when="+profiling")
```

Comment:
@white238, are you OK with this change? I prefer having the fixed version so that we don't get surprise updates. I guess I would need to rebuild the pre-built libraries on LC?

Reply:
yes I prefer fixed versions.. Thanks for catching this...